docker/openproblems-python-batch-integration/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM singlecellopenproblems/openproblems-r-base:latest
+
+USER root
+WORKDIR /
+
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+RUN sudo sed -i '$ d' /etc/apt/sources.list
+RUN sudo apt-get update --allow-releaseinfo-change && apt-get -y install gcc git python3-llvmlite
+
+# Install Python packages
+COPY ./docker/openproblems-python-batch-integration/requirements.txt ./requirements.txt
+RUN sudo pip install --no-cache-dir -r requirements.txt
```

Comment:
might be good to remove the apt-get libs after installation & update to make the docker image lighter.